src/components/Header/Header.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { useHistory } from 'react-router';
+import axios from 'axios';
+import UserContext from '../../Context/UserContext';
+import FilterableList from '../FilterableList/FilterableList';
+import Button from '../Button/Button';
+import MenuList from '../MenuList/MenuList';
+import styles from './Header.scss';
+
+
+
+function Header() {
+  const [isVisible, setIsVisible] = useState(false);
+  const [cities, setCities] = useState([]);
+  const [inputValue, setInputValue] = useState('');
+  const history = useHistory();
+
+  const changeFilterableListProperties = () => {
+    setIsVisible(isVisible => !isVisible);
+    setCities([]);
+  };
+
+  const context = useContext(UserContext);
+  const token = context.token;
+
+  useEffect(() => {
+    const cancelTokenSource = axios.CancelToken.source();
+    if (token !== '' && inputValue !== '') {
+      try {
+        axios
+          .get(`https://pfa.foreca.com/api/v1/location/search/${inputValue}`, {
+            headers: {
+              Authorization: `Bearer ${token}`
+            },
+            cancelToken: cancelTokenSource.source
+          })
+          .then(res => {
+            const dataCities = res.data.locations.map(city => {
+              return {
+                name: `${city.name.toLowerCase()}`,
+                country: `${city.country.toLowerCase()}`,
+                id: city.id
+              };
+            });
+            setCities(dataCities);
+          });
+      } catch (error) {
+        history.push('/error');
+      }
+    }
+    return () => cancelTokenSource.cancel();
+  }, [inputValue]);
+
+  const changeValue = event => {
+    setInputValue(event.target.value.toLowerCase());
+  };
+
+  return (
+    <header className={styles.header}>
+      <nav className={styles.container}>
+        <Button onClick={changeFilterableListProperties} children="Cities" />
```

Comment:
children is something that you usually pass in between opening and closing tags. 
```
 <Button onClick={changeFilterableListProperties} >
    "Cities"
 </Button>   
```


Moreover what exactly would user think when seeing a button, that says "Cities"? Honestly saying I, personally, not feel myself expecting something specific to happen.